### PR TITLE
Lpd 24185

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-24185

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -7357,6 +7357,8 @@ feature.flag.LPD-16469.description=Users have the ability to bulk manage permiss
 feature.flag.LPD-16469.title=Bulk Manage Permissions for Web Content Articles
 feature.flag.LPD-21265.description=Users can set default permissions for new web content (pages, blog posts, etc.) during the creation process.
 feature.flag.LPD-21265.title=Default Permissions
+feature.flag.LPD-21636.description=Users can define and manage a Picklist to configure Product Specification value.
+feature.flag.LPD-21636.title=Configure Specification value with Picklist
```
Comment (smottal) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: value should not be values? (in both the sentences)

**Review by smottal — COMMENTED** [review]
Just a thing on the locale but ... mine are always wrong :) so...

**Review by smottal — APPROVED** [review]

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (andrea-ale-sbarra) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: actually the value for the specification is only one

Comment (andrea-ale-sbarra) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: @JamesAGarcia can you please have a look at this? Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**